files/default/neutron-ha-tool.py

```diff
@@ -21,16 +21,16 @@
 #   2 - router migration required (with --l3-agent-check)
```

Comment:
Thanks, but you added the URL to a different commit.  I was suggesting to add it to this commit ("neutron-ha-tool: order imports") because the commit message is:

> Ordering imports makes more easy backport patches from the
> ansible playbook repository.

so anyone reading it will immediately wonder which repo that is.